mudirac/mudirac.xml

```diff
@@ -212,6 +218,9 @@ spec_expdec: $spec_expdec
         <data name="spectrum" label="Spectrum (${element})" format="png" from_work_dir="plots/input.spec.dat.png">
             <filter>generate_spectrum['generate_spectrum_selector']=='true'</filter>
         </data>
+        <data name="spectrum_raw_data" label="Raw data file of ${tool.name} (${element}) spectrum" format="tabular" from_work_dir="plots/spectrum.csv">
+            <filter>generate_spectrum['generate_spectrum_selector']=='true' and generate_spectrum['spectrum_raw_data']==True</filter>
```

Comment:
@patrick-austin  it didn't work as the galaxy is passing  `' '`  when  spectrum_raw_data is false, which causes invalid argument.

Reply:
The validator code is Pythonic right? if the two options are `"-f"` and `""` then maybe could get away with just `generate_spectrum['generate_spectrum_selector']=='true' and generate_spectrum['spectrum_raw_data']` since the former is truthy and the latter isn't (I haven't actually tried any of these suggestions, just going off memory of what we can/do do elsewhere)? I know we definitely use the true/falsevalue attributes elsewhere but I can't see it used with validators anywhere so maybe it's not workable. If you've tried a few options and none of them work we can leave it as is.